openid-sharedsignals-framework-1_0.md

```diff
@@ -1123,6 +1140,19 @@ description
   This is useful in multi-stream systems to identify the stream for human actors. The
   transmitter MAY truncate the string beyond an allowed max length.
 
+stream_ttl
+
+> **Transmitter-Supplied**, OPTIONAL. The refreshable time-to-live (TTL) of the stream in seconds, after which the Transmitter MAY either pause or disable the stream if it has not received any Receiver-initiated communication (defined below) in that duration. The syntax is the same as that of {{EXPIRES_IN}}.
+>
+> For streams created with the PUSH {{RFC8935}} delivery method, the Transmitter MUST refresh the TTL whenever:
+> * The Receiver calls any endpoint in the Event Stream Management API ({{management}}).
+>
+> For streams created with the POLL {{RFC8936}} delivery method, the Transmitter MUST refresh the TTL whenever:
+> * The Receiver polls the Transmitter for events.
+> * The Receiver calls any endpoint in the Event Stream Management API ({{management}}).
+>
+> If the Transmitter decides to update the stream, it MUST send a Stream Updated Event to the Receiver as described in {{status}}.
```

Comment:
```suggestion
> If the Transmitter decides to pause or disable the stream, it MUST send a Stream Updated Event to the Receiver as described in {{status}}.
```
It took me a second to understand why this sentence was here. Specifying pause or disable connects it to the language above.